lunavl/sdk/luna_faces.py

```diff
@@ -90,6 +124,26 @@ def __init__(self, coreDetection: Face, image: VLImage, estimatorCollection: Fac
         self._credibilityCheck: Optional[CredibilityCheck] = None
         self.estimatorCollection: FaceEstimatorsCollection = estimatorCollection
 
+    @property
+    def detectionSettings(self) -> VLFaceDetectionSettings:
+        """
+        Get current detection settings
+
+        Returns:
+            (VLFaceDetectionSettings) Current detection settings
+        """
+        return self._detectionSettings
+
+    @detectionSettings.setter
+    def detectionSettings(self, detectionSettings: VLFaceDetectionSettings):
+        """
+        Set new detection settings
+        Args:
+            detectionSettings (VLFaceDetectionSettings): new detection settings
+        """
+        self._mask = None
```

Comment:
this change the output of method `asDict`. It is bad. I am not sure that  the detectionSettings setter is necessary.